pkg/maker/buildcontent.go

```diff
@@ -914,3 +914,12 @@ func (m *Maker) ExecutesCommands(executes []Executes) string {
 	}
 	return content
 }
+
+func (c *Cbuild) GetAPIFiles(id string) []Files {
```

Comment:
```suggestion
func (c Cbuild) GetAPIFiles(id string) []Files {
```
Since the object is not changed. It would be safer to pass as object.

Reply:
However `Cbuild` is a large struct, the pointer is potentially more efficient, and especially for consistency I would rather use the same receiver for all functions which in this project is `(c *Cbuild)`.